src/i18n.js

```diff
@@ -0,0 +1,5 @@
+const NextI18Next = require("next-i18next").default;
+
+module.exports = new NextI18Next({
+    defaultLanguage: "en",
+});
```

Comment:
Is this syntax required? Could we use the import/export syntax we usually use?
```suggestion
import NextI18Next from "next-i18next";

export default new NextI18Next({
    defaultLanguage: "en",
});
```

Reply:
Yes this syntax is required (I double checked) due to some polyfill issues. I can't find the link any more that describes the issue but I can confirm `import` syntax won't work.